datadog_lambda/tracing.py

```diff
@@ -8,15 +8,17 @@
 
 from aws_xray_sdk.core import xray_recorder
 from aws_xray_sdk.core.lambda_launcher import LambdaContext
-
+ 
 from datadog_lambda.constants import (
     SamplingPriority,
     TraceHeader,
     XraySubsegment,
     TraceContextSource,
 )
 from ddtrace import tracer, patch
+from ddtrace import __version__ as ddtrace_version
 from ddtrace.propagation.http import HTTPPropagator
+from datadog_lambda import __version__ as lambda_version
```

Comment:
`lambda_version` can be a little bit misleading, probably rename to `datadog_lambda_version`?

Reply:
renamed correctly